v2.0/app/api/ui/superAdminMenu.js

```diff
@@ -236,5 +214,27 @@ module.exports = [
     name: 'New Super admin',
     icon: 'ios-add-circle',
     link: '/admin/add-super-admin'
-  }
+  },
+  {
+    key: 'tags-collections',
+    name: 'Tag Collections',
+    icon: 'md-reorder',
+    child: [
+      {
+        key: 'about-categories',
+        name: 'Tags & Tag Collections',
+        title: true
+      },
+      {
+        key: 'add-vendor',
```

Comment:
are we sure we want to put add-vendor as the key here?  maybe add-category?

Reply:
Your right.  I've fixed that.  (Not my bug:)  I don't know what that key is used for really, but maybe it needs to be unique.
Is the file menu.js (which also had the bug) obsolete?  I assume so and will delete it.